script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -229,12 +235,12 @@ These vampires are heretics that your average Executor wouldn't even stand a cha
 [sha:794d06828e2d89c0f332e18ad257b01bece1149a]{
 -- Page 591, Offset 18055.
 -- 貴族として自らの意志を許された吸血鬼」
-and are allowed to exercise their will as members of the aristocracy."
+aristocrat vampires who are allowed to exercise their own will."
```

Comment:
I kinda like this change. **Members of the** aristocracy almost hearkens back to shiki's misconception about vampires all being counts in old world castles. In other words, the old line makes it sound a bit like they're human aristocrats rather than being fancy _vampires_ on top of whatever social status they have in the Inter-Species Society

Reply:
1. Was the bolded `Members of the` supposed to be a change suggestion and
2. I don't love the comma splice between the prev line and here, suggest
> These vampires are heretics that your average Executor wouldn't even stand a chance against―――
> Aristocrat vampires who are allowed to exercise their own will."